gocd/generated-pipelines/relay-next-us.yaml

```diff
@@ -0,0 +1,175 @@
+format_version: 10
+pipelines:
+  region-deploy-relay-next-us:
+    environment_variables:
+      SENTRY_REGION: us
+    group: relay-next-region-deployments
+    lock_behavior: unlockWhenFinished
+    materials:
+      relay_repo:
+        branch: master
+        destination: relay
+        git: git@github.com:getsentry/relay.git
+        shallow_clone: true
+      upstream_pipeline:
+        pipeline: deploy-relay-next
+        stage: pipeline-complete
+    stages:
+      - ready:
+          jobs:
+            ready:
+              tasks:
+                - exec:
+                    command: true
+      - wait:
+          approval:
+            type: manual
+          jobs:
+            wait:
+              tasks:
+                - exec:
+                    command: true
+      - checks:
+          fetch_materials: true
+          jobs:
```

Comment:
And if checks are passing we want processing relay (which is `deploy-production` step) proceed automatically.

Reply:
This was added as a precaution while initially rolling this out, but removed as it should be safe.